.tool-versions

```diff
@@ -9,5 +9,5 @@
 # a more complete list, review the GitHub workflow to see the versions tested
 # in CI.
 
-golang 1.19.3
+golang 1.20.2
```

Comment:
@matthewmueller Should we already increase the default Go version with this PR as its working right now? What do you think?

Reply:
Yep sounds good!